app/src/main/res/values/strings.xml

```diff
@@ -154,10 +154,11 @@
     <string name="popup_playing_append">Queued on popup player</string>
     <string name="c3s_url" translatable="false">https://www.c3s.cc/</string>
     <string name="content">Content</string>
-    <string name="show_age_restricted_content_title">Age restricted content</string>
-    <string name="video_is_age_restricted">Show age restricted video. Future changes are possible from the settings.</string>
-    <string name="youtube_restricted_mode_enabled_title">YouTube restricted mode</string>
-    <string name="restricted_video">This video is age restricted.\n\nTurn on \"Age restricted content\" in the settings if you want to see it.</string>
+    <string name="show_age_restricted_content_title">Show age restricted content</string>
+    <string name="show_age_restricted_content_summary">Show content with an age limit (like 18+) and thus being probably unsafe for children.</string>
```

Comment:
```suggestion
    <string name="show_age_restricted_content_summary">Show content only suitable for ages 18 and above.</string>
```
@opusforlife2 Age restricted content is an addition, not an exclusive filter (?)

Reply:
The "only" there is ambiguous.

Reply:
How so?

Reply:
In the middle, a user might get confused which part it applies to. If it was "and above only" or "Only show content" then they would know clearly either way.

Reply:
@opusforlife2 "And above only" is not the same statement as "Show content only suitable __", because if it was it wouldn't be English. Insofar that it isn't, you already pointed out it means _a limitation to 18+ content_. Similarly, "Only show content __" is a valid statement that means _a limitation to 18+ content_, but now in functional English.
Still confused as to what the ambiguity is.

Reply:
When I say a user might get confused, I'm thinking of users for whom English is not the first language, but they still use the app in English. They might find it difficult to tell if that "only" applies to "show content" or to "ages 18 and above". Putting the "only" on either side would reduce chances of that confusion.

Anyway, I suggested another way to phrase it, above. Check it out.

Reply:
Only→suitable for, suitable for→ ages 18 and above
It isn't the simplest sentence, and quite possibly not the simplest sentence possible.
Ruining the actual content for people that speak English, in their own language, is not really why there are translations, or what should be done. Will reply.

Reply:
Fair enough. What about:
```suggestion
    <string name="show_age_restricted_content_summary">Show content restricted to ages 18 and above.</string>
```
?

Reply:
Good.